mirrors.d/mirror.ox.ac.uk.yml

```diff
@@ -0,0 +1,16 @@
+---
+name: mirror.ox.ac.uk
+address:
+  http: http://mirror.ox.ac.uk/sites/repo.almalinux.org/
+  https: https://mirror.ox.ac.uk/sites/repo.almalinux.org/
+  ftp: ftp://mirror.ox.ac.uk/sites/repo.almalinux.org/
+  rsync: rsync://mirror.ox.ac.uk/sites/repo.almalinux.org/
+geolocation:
+  country: UK
+  state_province: Oxfordshire
+  city: Oxford
+update_frequency: 6h
+sponsor: University of Oxford
+sponsor_url: https://www.ox.ac.uk
+email: unix-team-comms@it.ox.ac.uk
+private: true
```

Comment:
With `private` set to `true` but no `asn` or `subnets` defined the mirror won't actually do anything or be used.

Do you want to force network traffic within your subnet(s) and/or ASN to your mirror first (will fall back to other mirrors if yours is down unless you set `monopoly` to `true`)?  Do you want the mirror to be public or private?

Reply:
I've obviously misunderstood the meaning of 'private'.

Now hopefully set to 'false'